lib/install/management/finder_test.go

```diff
@@ -120,8 +121,10 @@ func testSearchVCHs(ctx context.Context, t *testing.T, v *validate.Validator, ex
 			ref := vm.Reference()
 			svm := simulator.Map.Get(ref).(*simulator.VirtualMachine)
 
+			template := config.VirtualContainerHostConfigSpec{}
+			keys := extraconfig.CalculateKeys(template, "ExecutorConfig.ExecutorConfigCommon.ID", "")
```

Comment:
Unrelated to this PR, but: It seems like it would be helpful to have a `extraconfig.CalculateKey` helper that calls `extraconfig.CalculateKeys`, verifies that a single key is returned, and returns the single value to the caller. This would avoid the need to follow the `extraconfig.CalculateKeys` call with an unchecked `[0]` all over the place.